README.md

```diff
@@ -1,24 +1,215 @@
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li>Deployed on Render <a href="https://rails-b62y.onrender.com/">Capstone-Back-End</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Franwork</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the repository.
+
+- **Was created with RoR** 
+- **Has a login and registration authentication system with the help of devise**
+- **Has a database that stores the records of it's rooms, branches, users and reservations**
+- **Users have roles and only admins can manipulate the rooms and braches**
+- **Has a private api endpoint that allows users to see their reservations and rooms**
+- **Has Rubocop to enforce best ruby practices**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Follow these steps in order to run ruby code.
+
+### Prerequisites
+
+First you need to ensure that you have ruby installed in your computer. 
+
+### Install
+
+It varies depending on your operating system so here are some links to help you install it on [Windows](https://gorails.com/setup/windows/10) and on [Ubuntu](https://www.ruby-lang.org/en/documentation/installation/#apt). 
+
+For MacOS run the following commands
+
+```sh
+  brew install rbenv ruby-build
+  # Add rbenv to bash so that it loads every time you open a terminal
+  echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+  source ~/.bash_profile
+
+  # Install Ruby
+  rbenv install 3.0.1
+  rbenv global 3.0.1
+  ruby -v
+```
+
+This example shows how to install Ruby 3.0.1 which was the latest version in April 2021, but you can check to see if there is a newer version [here](https://www.ruby-lang.org/en/downloads/releases/). Iy is also assumed that you have [homebrew](https://brew.sh/) already installed on your Mac.
+
+Next you'll want to intall Rails and postgreSQL by running the following commands, but you can also use any SQL application of your choice
+
+```shell
+  gem install rails -v 7.0.6
+
+  sudo apt install postgresql libpq-dev
+```
+
+And start postgreSQL and the app with 
+
+```shell
+  sudo service postgresql start
+
+  rails server
+```
+
+### Setup
+
+Once you have ruby installed, run this command to get the project on your local machine.
+
+```sh
+  git clone git@github.com:hernandanielzamora/Capstone-Back-End.git
+```
+
+### Usage
+
+Once you have the project installed, access the project's root directory usign this command 
+
+```sh
+  cd Capstone-Back-End
+```
+
+And lastly, run this command to see the project in in your browser.
+
+```sh
+  rails server
+```
```

Comment:
- [ ] When I try to create the database, this happens:

![image](https://github.com/hernandanielzamora/Capstone-Back-End/assets/57421823/1ca2c412-fd4a-4084-bf48-eed76604a0ae)

This is because the master key is absent.  Therefore, your users will need to create their own following the next instructions:

```text
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://github.com/hernandanielzamora/Capstone-Back-End/assets/57421823/32f334f5-ad6b-4e0b-9ef2-281caddfcca3)

Kindly, include these instructions on your readme so your users are able to create the database. 👍